packages/framework-integration-tests/integration/providers/kubernetes/deployment/deployment.integration.ts

```diff
@@ -26,6 +29,19 @@ describe('Kubernetes provider', () => {
       return item?.metadata?.name
     })
 
+    pods.body.items.map(async (pod: Kubernetes.V1Pod) => {
```

Comment:
I'm wondering if this kind of things can be extracted on a manager to be reused or simply use the K8sManager from the main project. I think that there are some tasks that we want to perform inside the K8s cluster during tests. What do you think?

Reply:
I think I'm using a manager for tests since the current one is in the provider-infrastructure package and I would have to move it to `dependencies` to use it. Thanks @rdiaz82 !